opm/simulators/utils/PartiallySupportedFlowKeywords.cpp

```diff
@@ -230,85 +239,86 @@ partiallySupported()
          {
             "TRACER",
             {
-               {4,{false, allow_values<std::string> {}, "TRACER(SOLPHASE): partitioned tracer model not supported - ignored as not used"}}, // SOLUTION_PHASE
-               {6,{false, allow_values<std::string> {}, "TRACER(ADSPHASE): partitioned tracer model not supported - ignored as not used"}}, // ADSORB_PHASE
+               {1,{true, [](const std::string& val){ return val.size()<=3;}, "TRACER(NAME): Only names of tracers up to 3 characters are supported."}},
+               {4,{true, allow_values<std::string> {}, "TRACER(SOLPHASE): partitioned tracer model not supported use default"}}, // SOLUTION_PHASE
+               {6,{true, allow_values<std::string> {}, "TRACER(ADSPHASE): partitioned tracer model not supported use default"}}, // ADSORB_PHASE
             },
          },
          {
             "TRACERS",
             {
                {5,{false, allow_values<std::string> {"NODIFF"}, "TRACERS(DIFFOPT): numerical diffusion control not implemented. - ignored as not used"}}, // NUMERIC_DIFF
```

Comment:
Suggest to stop for anything else than NODIFF (until implemented), since the tracer solution will change depending on this setting.

Reply:
> What abut the command line parameter `--ecl-strict-parsing`, is this still useful?

I thing this is something different. This is more related to  [ParseContext](https://github.com/OPM/opm-common/blob/a154a2a49f696d30e617285c8d9897ae8714d8ae/src/opm/input/eclipse/Parser/ParseContext.cpp#L73)

missing slashes and so on. Personally I haven't used this option ever as far as I can remember. 

When/If this PR goes in, I think it would be useful with a command option which will accept all keyword (ignore the critical variable) and just run until something crashes (or in some cases the simulator will actually run).